ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -342,6 +357,15 @@ const Annotations = React.memo(({ variant, mainGeneId, showMainGene }) => {
       ).map(e => <div key={e}>{e}</div>)]}
       <VerticalSpacer height={5} />
       <VariantLocusListLabels variant={variant} familyGuids={variant.familyGuids} />
+      {variant.highConstraintRegion && (
+        <span>
+          <HorizontalSpacer width={12} />
+          <Popup
+            trigger={<Label color="red" horizontal size="tiny">High Constraint Region</Label>}
+            content="The variant locate in one of the high constraint region"
```

Comment:
there doesn't seem to be any additional information provided in the popup, so I would just make this a plain label